torch/_dynamo/utils.py

```diff
@@ -1630,6 +1630,25 @@ def numpy_attr_wrapper(obj, name):
         return numpy_to_tensor(out)
 
 
+class numpy_method_wrapper:
+    """Convert obj from torch.Tensor to torch_np.ndarray and call method. Then convert result back to torch.Tensor."""
+
+    def __init__(self, method: str):
+        self.method = method
+        self.__name__ = "wrapped_" + self.method
+
+    def __repr__(self):
+        return f"<Wrapped method <original {self.method}>>"
+
+    def __call__(self, *args, **kwargs):
+        obj = args[0]
+        if isinstance(obj, torch.Tensor):
+            obj = torch_np.ndarray(obj)
```

Comment:
Why just the first variable? Shouldn't you do do the same for all the tensor variables? What if there are tuples of tensors?

Reply:
So in torch_np they will cast all tensor like arguments to torch_np.ndarray implicitly. I can add some tests.